fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulktools/Main.java

```diff
@@ -175,50 +223,81 @@ private int getFhirResourceNumberFromBufferReader(BufferedReader resReader) thro
 
 
 
-    private void writeFhirResourceFromBufferReader(BufferedReader resReader,  AmazonS3 cosClient, ByteArrayOutputStream bufferStream, int numOfRes4Seg) throws Exception {
+    private boolean writeFhirResourceFromBufferReader(BufferedReader resReader,  AmazonS3 cosClient, ByteArrayOutputStream bufferStream, long num4Seg, boolean isSegBySize) throws Exception {
         int lineRed = 0;
         int segNum = 0;
+        long segSize = 0;
+        HashSet<String> UniqueResourceIds= new HashSet<String>();
```

Comment:
use the diamond operator
```suggestion
        HashSet<String> UniqueResourceIds= new HashSet<>();
```

Reply:
mixed usage with line 232